### DISCUSSION TITLE
Software (GUI) to create/edit JSBSim models

### DISCUSSION THREAD

**Discussion by demonixis** [JSBSim-Team/jsbsim#890] [Q&A]
Hello,
I'm looking for a gui editor for JSBSim models. I've found JSBSim commander, but the last release is from 2006 and it doesn't open today's models. Do you know other alternatives?

**Comment by seanmcleod** [comment]
Not that I'm aware of, haven't come across any others, but haven't specifically looked for any.

Over the years there have been a couple of requests for an updated JSBSim Commander and/or suggestions of updating it, but no-one has got round to doing it yet 😉 

https://github.com/JSBSim-Team/jsbsim/issues/279#issuecomment-619623384

https://github.com/JSBSim-Team/jsbsim/issues/279#issuecomment-622455101

https://github.com/JSBSim-Team/jsbsim/issues/273#issue-596340011

https://github.com/JSBSim-Team/jsbsim/issues/273#issuecomment-624688860

**Reply by demonixis** [reply]
Ok thanks for the infos. I don't know my combined skills in c++, JSBSim and WXWidget can really help. However it's probably possible to build something with something else.

I started to work on an app that help to create helicopter from the one in JSBSim.

**Comment by BudDavis** [comment]
I have been messing with getting jsbsimcommander to compile with a linux distribution from 2023.  Not for the faint of heart; it is also 32 bit deep inside.

Does anyone have knowledge of the specific OS version that the github code would run under?  A running version would make porting this version, or replacing it a simpler endeavor.

thanks,
Bud Davis
